packages/ui-client/src/components/GenericMenu/GenericMenuItem.tsx

```diff
@@ -16,7 +15,7 @@ export type GenericMenuItemProps = {
 	variant?: string;
 };
 
-const GenericMenuItem = ({ icon, content, addon, status, gap, tooltip }: GenericMenuItemProps) => (
+export const GenericMenuItem = ({ icon, content, addon, status, gap, tooltip }: GenericMenuItemProps) => (
```

Comment:
Everything seems good, but since now we're using named exports, please remove the default export in the end of the file.

Reply:
<img src="https://media3.giphy.com/media/d6JZehe5QJGBzlUjgi/giphy.gif"/>

Reply:
@gabriellsh My bad, the convention _[(Item 4)](https://developer.rocket.chat/docs/conventions#components)_ is to use default exports for component modules. I adjusted the files accordingly, leaving named exports only for index files.

Reply:
My bad, I forgot we could export default exports as named exports using `as`.

It's better this way, thank you!